roles/bridge-setup/tasks/main.yml

```diff
@@ -1,9 +1,31 @@
 ---
 
-- name: Install bridge-utils
+- name: Install bridge-utils and iptables
   yum:
-    name: bridge-utils
+    name: "{{ item }}"
     state: latest
+  with_items:
+    - bridge-utils
+    - iptables
+    - iptables-services
+
+- name: enable iptables
+  service:
```

Comment:
Should also manage the services with systemd module, not the service module

Reply:
changed.